packages/sanity/src/core/studio/copyPaste/__test__/transferValue.test.ts

```diff
@@ -988,24 +1188,6 @@ describe('transferValue', () => {
       expect(targetValue.bio[0].children[0]._key).not.toEqual('someOtherKey')
     })
 
-    test('can copy array of numbers', async () => {
```

Comment:
Was this test moved elsewhere? 🤔

Reply:
I removed a duplicate - there is still a [test for this here](https://github.com/sanity-io/sanity/blob/b8d5fca8e0171b434afb1685296202b0957b3b06/packages/sanity/src/core/studio/copyPaste/__test__/transferValue.test.ts#L564)